applications/CoSimulationApplication/python_scripts/function_callback_utility.py

```diff
@@ -11,11 +11,33 @@
              'modf', 'pi', 'pow', 'radians', 'sin', 'sinh', 'sqrt',
```

Comment:
did you have to duplicate this because the included pyKratos ?

Reply:
yes and bcs the one in the core is designed to work with nodes, which does not apply in our case